gxawsserverless/src/main/java/com/genexus/cloud/serverless/aws/handler/LambdaEventBridgeHandler.java

```diff
@@ -0,0 +1,70 @@
+package com.genexus.cloud.serverless.aws.handler;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.genexus.cloud.serverless.Helper;
+import com.genexus.cloud.serverless.exception.FunctionRuntimeException;
+import com.genexus.cloud.serverless.model.EventMessage;
+import com.genexus.cloud.serverless.model.EventMessageResponse;
+import com.genexus.cloud.serverless.model.EventMessageSourceType;
+import com.genexus.cloud.serverless.model.EventMessages;
+import com.genexus.webpanels.WebUtils;
+import json.org.json.JSONObject;
+
+import java.util.Map;
+
+public class LambdaEventBridgeHandler extends LambdaBaseEventHandler implements RequestHandler<Map<String, Object>, String> {
+
+	public LambdaEventBridgeHandler() throws Exception {
+		super();
+	}
+
+	public LambdaEventBridgeHandler(String entryPointClassName) throws Exception {
+		super(entryPointClassName);
+	}
+
+	@Override
+	public String handleRequest(Map<String, Object> stringObjectMap, Context context) {
+		String jsonEventRaw = Helper.toJSONString(stringObjectMap);
+		if (logger.isDebugEnabled()) {
+			logger.debug("handleRequest started with event: " + jsonEventRaw);
+		}
+
+		String errorMessage;
+		EventMessageResponse response;
+
+		try {
+			EventMessages msgs = new EventMessages();
+			EventMessage msgItem = new EventMessage();
+			msgItem.setMessageSourceType(EventMessageSourceType.ServiceBusMessage);
+			if (stringObjectMap.containsKey("time")) {
+				msgItem.setMessageDate(WebUtils.parseDTimeParm(stringObjectMap.get("time").toString()));
+			}
+			msgItem.setMessageId(stringObjectMap.getOrDefault("id", "").toString());
+			if (stringObjectMap.containsKey("detail")) {
+				msgItem.setMessageData(new JSONObject(jsonEventRaw).getString("detail"));
+			}
+			for (Map.Entry<String, Object> entry : stringObjectMap.entrySet()) {
+				Helper.addEventMessageProperty(msgItem, entry.getKey(), entry.getValue().toString());
+			}
+			msgs.add(msgItem);
+			response = dispatchEvent(msgs, jsonEventRaw);
+		} catch (Exception e) {
+			errorMessage = "HandleRequest execution error";
+			logger.error(errorMessage, e);
+			throw new FunctionRuntimeException(errorMessage, e);
+		}
+
+		if (response == null) {
```

Comment:
It seems that response will never be null at this point

Reply:
I think it could be null if the GX Proc does not have out parameters. Will leave it just in ase.